lib/Zonemaster/CLI.pm

```diff
@@ -623,6 +623,10 @@ sub run {
         if ( $self->test and @{ $self->test } > 0 ) {
             foreach my $t ( @{ $self->test } ) {
                 my ( $module, $method ) = split( '/', $t, 2 );
+                if ( $module =~ /^([A-Za-z_]+)[0-9]+$/ ) {
```

Comment:
Why is underscore "_" included in the module name part? There is no such case and no plan for that.

Reply:
I think it's because I couldn't find any place defining the character set for the module name. Maybe I overlooked something.

Reply:
In https://github.com/zonemaster/zonemaster/blob/master/docs/internal/templates/specifications/tests/TestCaseIdentifierSpecification.md the test levels (modules) are defined and all matches /^[A-Z]+$/i.

Reply:
Updated